Terran-One/cw-simulate-ui
# Show Balance in Summary.

## Issue link

[WL-XXX](https://trello.com/c/Ribsjw0o/144-show-contract-balances-in-summary)

## Description
Contract and Sender balance are now shown under Summary tab.

## Test steps

1. Go to simulation page.
2. Perform some executions.
3. Now traverse over steps and see if the balance shown is changing according to step in summary tab.
